src/api/app/controllers/webui/projects/pulse_controller.rb

```diff
@@ -2,7 +2,7 @@ module Webui
   module Projects
     class PulseController < WebuiController
       before_action :set_project
-      before_action :set_pulse, if: -> { request.xhr? }
+      before_action :set_pulse
```

Comment:
Why do you remove this? Now you load data twice. Once from the request (it also blocks it) and once via ajax when the page has finished rendering.

Loading the data after rendering is a feature :smile:

Reply:
I only need the `@range` for the view, so I'll revert this change and have another `before_action :set_range` instead.

I went a bit overboard with this (I was probably annoyed after fixing some feature specs hahaha, my bad!!!)